crates/matrix-sdk-base/src/latest_event.rs

```diff
@@ -35,6 +37,10 @@ pub fn is_suitable_for_latest_event(event: &AnySyncTimelineEvent) -> PossibleLat
             PossibleLatestEvent::YesMessageLike(message)
```

Comment:
Should this perhaps be renamed to `YesRoomMessage` ?
After all Polls are still falling under `MessageLike` in ruma even though they are not of type m.room.message

Reply:
Yeah. Also `SyncMessageLikeEvent<RoomMessageEventContent>` has a type alias `SyncRoomMessageEvent` that we should use, same for the poll one.

Reply:
Cool, done!